bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/container.jsx

```diff
@@ -41,6 +44,6 @@ export default withTracker(() => {
     dispatchTogglePresentationDownloadable,
     isOpen: Session.get('showUploadPresentationView') || false,
     selectedToBeNextCurrent: Session.get('selectedToBeNextCurrent') || null,
-    isPresenter: PresentationService.isPresenter('DEFAULT_PRESENTATION_POD'),
```

Comment:
I think it would be best if you could change the service to correctly find presenter (or remove it but check where it was used - like in whiteboard service)